docs/language.rst

```diff
@@ -1757,6 +1757,60 @@ The syntax for calling external call is the same as the external call, except fo
 that it must be done on a contract type variable. Any error in an external call can
 be handled with :ref:`try-catch`.
 
+Internal calls and externals calls
+___________________________________
+
+An internal function call is one that is executed by the current contract. This
+is much more efficient than an external call. An external call requires the
+address of the contract to call, the arguments must be *abi encoded* (also known
+as serialization), then the runtime must set up the VM for the called contract
+(the callee). It must decode the arguments, and then encode return values. Lastly
+the first contract (the caller) must decode return values.
+
+Whenever a call is done on a contract type, this will intiate an external call.
+Note that ``this`` returns the current contract, so ``this.foo()`` will do an
+external call, which is much more expensive than ``foo()``.
+
+Passing accounts on Solana on external calls
+____________________________________________
+
+The Solana runtime allows you the specify the accounts to be passed for an
+external call. This is specified in an array of the struct ``AccountMeta``. This
```

Comment:
Is `AccountMeta` going to be a reserved word?

Reply:
We could do something like `SolanaSdk.AccountMeta`.

Reply:
Now you've made `SolanaSdk` a reserved word (and it's really ugly).

Reply:
So I've decided to bite the bullet and go ahead with a change I had planned for some time.

If we want to introduce new types, or builtin functions, we should hide them using solidity imports. The import system used by solidity mimics what javascript does. In javascript/node, builtins (apart from ancient onces) are hidden behind imports.

I've change the code and now there is an import called:

```
import 'solana';
```
Which imports `AccountMeta` and `AccountInfo`. You can do all the renaming tricks that import permits.

This keeps the namespace clean. 

On Solana we also have a `verifySignature()` call that should go into this import (something for another PR)